inc/saitypes.h

```diff
@@ -755,6 +758,9 @@ typedef struct _sai_qos_map_params_t
     /** Forwarding class */
     sai_uint8_t fc;
 
+    /** CFI value */
+    sai_uint8_t cfi;
```

Comment:
this change is not binary backward compatible

Reply:
this will not pass metadata checks, unless we decide manually to make this change go + @lguohan